plugins/source/aws/resources/services/apprunner/tag_fetch.go

```diff
@@ -12,6 +14,9 @@ import (
 func resolveApprunnerTags(path string) schema.ColumnResolver {
 	return func(ctx context.Context, meta schema.ClientMeta, r *schema.Resource, c schema.Column) error {
 		arn := funk.Get(r.Item, path, funk.WithAllowZero()).(*string)
+		if strings.Contains(aws.ToString(arn), "DefaultConfiguration/1/00000000000000000000000000000001") {
```

Comment:
I think this line might deserve a comment 😅

Reply:
Done